### DISCUSSION TITLE
Tasmota on ESP32C6 (specifically the SEEED ESP32C6 board

### DISCUSSION THREAD

**Discussion by scargill** [arendst/Tasmota#22165] [Support request] [answered]
Trivial I know, but I web flashed Tasmota (not any variation, just "tasmota" development today onto this tiny SEEED ESP32C6 board - partly as a test as I could not get WLED to work with the board at all). Great -  just the first TASMOTA option under development section.... up to now all is good but I have a weird one - I put a block of 10 ws2812 LEDS onto GPIO22 and once I realised the board's 3v3 supply would not handle that number of LEDS (unusual but then everything in the SEEED board is enclosed in metal) - I put the LEDs  onto 5V - perfect.  But when I tried Tasmota's SCHEME 11 (console command) to get RAINBOW, I got red and green lights. I switched to SCHEME12 and perfect rainbow. Is this a bug in the latest Tasmota dev, or me not using the right Tasmota option for the C6 ESP32, or just the C6 board? I definitely got the numbers right - just finished blogging it here. https://tech.scargill.net/seeed-wednesday/ if that helps - near the end - complete with short embedded video of the rainbow.

**Comment by sfromis** [comment, marked as answer]
No bug, just a quirk, as you did **not** get the numbers right.

When reading the docs for the [Scheme](https://tasmota.github.io/docs/Commands/#scheme) command, you need to pay attention to this part:
> **Note:** If using ESP32, or any build with the DDP feature (USE_NETWORK_LIGHT_SCHEMES), enabling DDP is Scheme 5 for plain RGB and Scheme 16 for addressable leds. Then the normal neopixel scheme numbers become 6-15 instead of 5-14

Hence, your scheme 11 for rainbow instead becomes scheme 10 for kwanzaa.
